src/content/docs/workflows/about-workflows/connect-repo-for-workflows.mdx

```diff
@@ -0,0 +1,45 @@
+---
+page_id: d360c480-49a8-47d3-b178-1abd34ae669f
+title: Connect your workflows repo and branch
+sidebar:
+  order: 2
+relatedArticles:
+  - 85c135db-f55a-4edf-8d1b-0af4140c1925
+  - d672fca7-6c6a-49f8-85c9-6e86ce99f440
+  - 8554d648-fc2f-4ef9-aaa9-4f27b3c05205 
+---
+
+We want you to keep control of your own code, in your own repo, so you can update and commit as usual. Then sync your code with one click and it’s ready to use.
+
+You first need to set up your repo and connect the branch to Kinde. 
+
+## Set up your repo
+
+For the code sync to work, your workflow files need to be stored in a specific place in your repo.
+
+1. In the root of the repo create a file called `kinde.json` with the content:
+    
+    ```jsx
+    {
+      "rootDir": "kindeSrc" // This is root folder Kinde will use to discover workflows.
+    }
```

Comment:
This tells Kinde where your custom code lives. If you prefer to use an alternative directory you can change `kindeSrc` to something else